boards/arm/bl5340_dvk/bl5340_dvk_cpuapp_common.dts

```diff
@@ -215,8 +219,8 @@
 		reset-gpios = <&gpio0 6 GPIO_ACTIVE_LOW>;
 		cmd-data-gpios = <&gpio0 12 GPIO_ACTIVE_LOW>;
 		rotation = <270>;
-		width = <240>;
-		height = <320>;
+		width = <320>;
+		height = <240>;
```

Comment:
From https://www.buydisplay.com/2-8-inch-tft-touch-shield-for-arduino-w-capacitive-touch-screen-module: `ER-TFTM028-4 is 240x320 dots 2.8" color tft lcd module display with` - has this been tested and works?

Reply:
The sample app hard faults without this change.  Maybe LV_Z_POINTER_KSCAN_SWAP_XY works differently than before?

Reply:
probably kscan pointing at the ft5336 driver that does not implement an API anymore, see my other comment, you can add a `zephyr,kscan-input` to make it work again and point `kscan_input` to that instead

Reply:
Has this been tested and working since the values are flipped?

Reply:
Yes. It has been tested with the lvgl sample app.